docs/02-skog-context.md

```diff
@@ -59,6 +59,19 @@ function skogMiddleware(req, res, next) {
 }
 ```
 
+Lastly, sometimes there is already a context opened when calling this middleware (e.g., there is another middleware that opnens such context). To support it, we call `getFields` and add the returned object to the fields:
```

Comment:
opnens -> opens

Perhaps swap "a context opened" for "a context created" and later "accesses such a context". You would open a file but create a context (object).

Reply:
I have replaced the word "open" with "create" but...

Technically is not the same to _create_ a context and to _(???)_ (not sure which verb should be used)

For example in React you have `React.createContext` to _create_ a context but then you have `YourContext.Provider`, a component that "???" (what should be the verb here?).

`YourContext.Provider` has two arguments: `value` and `children`.

- `value` in React is `fields` in Skog
- `children` in React is `fn` in Skog

The context is _created_ by Skog. What we export is a function that provides the value to the context. The context is _(???)_ by the user.

Reply:
Another option is just to omit the word context at all since it is an implementation detail and instead rename the function `runWithSkogContext` -> `runWithSkog` or something...